benchmarks/matrixBench.cpp

```diff
@@ -5,9 +5,12 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #include <benchmark/benchmark.h>
+#include <khiva/internal/matrixInternal.h>
 #include <khiva/matrix.h>
 #include <math.h>
-#include "khiva/matrixInternal.h"
+
```

Comment:
are these empty lines needed?, just asking, I'm not sure they follow the c++ coding style